content-src/components/Snippet/Snippet.js

```diff
@@ -0,0 +1,37 @@
+const React = require("react");
+const classNames = require("classnames");
+
+const Snippet = React.createClass({
+  onClose(e) {
+    e.preventDefault();
+    this.props.setVisibility(false);
+  },
+  render() {
+    const {visible, description, title, image} = this.props;
+    return (<div className={classNames("snippet", {"hidden": !visible})}>
```

Comment:
We'll save a bit of render/layout time, and follow the React idiom better if, instead of passing in ```visible```, simple have the parent not render <Snippet> at all if visible is set to false.